src/pages/likedVideos/LikedVideos.jsx

```diff
@@ -1,5 +1,30 @@
 import React from "react";
+import { ThumbnailCard } from "../../components/index-component";
 
 export const LikedVideos = () => {
-  return <div>LikedVideos</div>;
+  return (
+    <>
+      <h1 className="text-3xl capitalize text-center my-8 font-semibold">
+        liked videos
+      </h1>
+      <div className="flex justify-center py-8 flex-wrap gap-4">
+        <div className=" w-1/6 min-w-fit  bg-gray-800 rounded-lg overflow-hidden">
+          {" "}
+          <ThumbnailCard />
+        </div>
+        <div className=" w-1/6 min-w-fit  bg-gray-800 rounded-lg overflow-hidden">
+          {" "}
+          <ThumbnailCard />
+        </div>{" "}
+        <div className=" w-1/6 min-w-fit  bg-gray-800 rounded-lg overflow-hidden">
+          {" "}
+          <ThumbnailCard />
+        </div>{" "}
+        <div className=" w-1/6 min-w-fit  bg-gray-800 rounded-lg overflow-hidden">
+          {" "}
+          <ThumbnailCard />
+        </div>
```

Comment:
Suggestion: instead of writing this multiple times, if this is in progress write a comment here

Reply:
This page is yet to complete, already mentioned `files to check`